app/assets/javascripts/services/miq_service.js

```diff
@@ -158,4 +158,10 @@ ManageIQ.angular.app.service('miqService', ['$timeout', '$document', '$q', 'API'
       miqService.sparkleOff();
     }
   };
+
+  this.resetData = function(vm, hshName, scope) {
+    vm[hshName] = angular.copy( vm.modelCopy );
```

Comment:
`hshName` is not needed, `vm.model` always contains the name, so `vm[vm.model]` will work.